spec/core/data_structures.md

```diff
@@ -455,8 +466,8 @@ func SumTruncated(bz []byte) []byte {
 
 | Name          | Type   | Description                   | Field Number |
 |---------------|--------|-------------------------------|--------------|
-| message_delay | [google.protobuf.Duration](https://developers.google.com/protocol-buffers/docs/reference/google.protobuf#google.protobuf.Duration) | Bound for how long a proposal message may take to reach all validators on a newtork and still be considered valid. | 1            |
-| precision     | [google.protobuf.Duration](https://developers.google.com/protocol-buffers/docs/reference/google.protobuf#google.protobuf.Duration) | Bound for how skewed a proposer's clock may be from any validator on the network while still producing valid proposals. | 2            |
+| precision     | [google.protobuf.Duration](https://developers.google.com/protocol-buffers/docs/reference/google.protobuf#google.protobuf.Duration) | Bound for how skewed a proposer's clock may be from any validator on the network while still producing valid proposals. | 1            |
+| message_delay | [google.protobuf.Duration](https://developers.google.com/protocol-buffers/docs/reference/google.protobuf#google.protobuf.Duration) | Bound for how long a proposal message may take to reach all validators on a newtork and still be considered valid. | 2            |
```

Comment:
I swapped them according to what was reflective to the structure in the code see https://github.com/tendermint/tendermint/blob/27c523dccbe7f2f577868ee871b0042c1fef9a53/types/params.go#L85 @cmwaters

Reply:
Right, your reference should actually be the proto file:

https://github.com/tendermint/tendermint/blob/27c523dccbe7f2f577868ee871b0042c1fef9a53/proto/tendermint/types/params.proto#L78

That's the holy grail

Reply:
Protobuf is what we use for serialization but it's also, in a way, the language of the spec. If someone wanted to write Tendermint in Rust the specification requires them to adhere to the protobuf definitions (so that two implementations could talk to each other for example).